def test_function(): #создали функцию
    print('5')        # это я для себя, что бы видеть что она работает
    def inner_fuction(): #внутри создали еще функцию
        print('Я в области видимости функции test_function') #функция будет выводить в консоль строку

    #inner_fuction() #если мы тут вызовем функцию, и у на не будет вызвана первая функция,
                     # ничего не произойдет


test_function() #если следом вызвать первую функцию, то в консоль выйдет число 5 и строка


inner_function()  #если попробовать даже после всех вызовов вызвать еще раз внутреннюю ф-ю
                   #выдаст ошибку
